doc/changes/0.2.rst

```diff
@@ -0,0 +1,18 @@
+.. _changes_0_2:
+
+Version 0.2 (in progress)
+------------------------
+
+- Accelerated block coordinate descent solver with working sets for problems with group penalties (:gh:`29`, :gh:`28`, :gh:`26`)
+
+- Proximal Newton solver with working sets for problems with non-quadratic datafits (:gh:`51`)
```

Comment:
can you put the name of the class here (as for datafits?). It guides the user to the correct class

Reply:
even better if you manage to link to the doc of the corresponding class...

Reply:
we should :)